content/millennium/dstu2/care-provision/care-plan.md

```diff
@@ -53,7 +55,7 @@ Notes:
 <%= headers status: 200 %>
 <%= json(:dstu2_care_plan_bundle) %>
 
-<%= headers 200, {Functionality: 'care plan with participants'} %>
+<%= headers 200, {Functionality: 'CareTeam (CarePlan with participants)'} %>
```

Comment:
I believe this needs to be `headers status:200, head: {...}` (and also, I'll be changing this with the template, that list was intended to represent request headers originally :)